report/3.Theory/SemanticsDescription.tex

```diff
@@ -451,27 +505,29 @@ \subsection{Big-Step Semantic for Expressions}
 
 \subsection{Big-Step Semantic for Statements}
 Statements $Stm$ can change the store, because they can change the value of an existing variable.
-The transition-system for $stm$ is $(Stm\ \Cross\ Sto\ \cup Sto, \rightarrow_{stm}, Sto)$.
-Given the variable bindings $env_V$ and the procedure bindings $env_P$, the execution of the statement $S$ will 
+Because of the fact that the category have rewriting rules for declarations, this means that statements can also both change the variable environment $Env_V$ and the function environment $Env_F$.
```

Comment:
Måske skal sætningen i stedet være?
"Because the category has rewriting rules for declarations, stratements can also change both the variable environment $Env_V$ and the function environment $Env_F$."
Syntes det er en bedre, kortere og mere konsis (?) sætning, og så giver første del af den faktisk også mening

Reply:
Jo egentlig. Fixed